server/models/index.js

```diff
@@ -1 +1,76 @@
-index.js
\ No newline at end of file
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+mongoose.connect('mongodb://localhost/hunch');
+
+
+const UserSchema = new Schema({
+    name: {
+        type: String
+    },
+    points: {
+        type: Number
+    },
+    answer: { 
+        type: Schema.Types.ObjectId,
+        ref: 'Answer'
+    }
+})
+
+const TopicSchema = new Schema({
+    topic: {
+        type: String
+    }
+})
+
+const GameSchema = new Schema({
+    currentTopic: { 
+        type: Schema.Types.ObjectId,
+        ref: 'Topic' },
+    round: {
+        type: Number
+    },
+    users: [UserSchema],
+    topics: [TopicSchema]
+})
+
+const SelectionSchema = new Schema({
+    guessing_user: {
```

Comment:
I changed things around since the last time we talked about this so def understand your confusion.

Before we had planned on doing something like this with the mixed type for answers: { user id: answer string }, my thinking was that answers needed to be stored in the database so that we can match them to users and selections correctly with unique IDs instead of strings.

For each topic/round, a user gives a single answer. For each answer, there should be as many selections as there are users, where users (guessing user) will guess who said the answer (suspected user).
At the end of each topic/round, people's guesses are displayed as well as who actually said what answer. Points are added, so there's need to hold on to answers or selections anymore and they can be cleared for the next topic/round (why each user has only one answer as opposed to an array of answers). There are probably a million better ways to do this though and would like your input.

That was just some clarification, besides that I agree that selections don't need to be turned into a model if we keep things the way they are.